web/sites/default/settings.php

```diff
@@ -53,4 +53,6 @@
   if (is_array($settings)) {
     $settings['trusted_host_patterns'] = array('^'. preg_quote($primary_domain) .'$');
   }
-}
\ No newline at end of file
+}
+
+$settings['hash_salt'] = "testtest";
```

Comment:
@kxwang, you accidentally included a change to the settings.php. This value should be set in your settings.local.php to ensure that you don't need to update a file that can accidentally get pushed to production.